inc/namespace.php

```diff
@@ -954,3 +976,95 @@ function set_wp_debug_constants() : void {
 	}
 	define( 'WP_DEBUG_DISPLAY', false );
 }
+
+/**
+ * Returns true when environment is running in the cloud. Returns false in all
+ * other conditions, such as local-server or local-chassis.
+ *
+ * @return bool
+ */
+function is_cloud() : bool {
+	return in_array( Altis\get_environment_architecture(), [ 'ec2', 'ecs' ], true );
+}
+
+/**
+ * Returns a PSR-3 compatible logger.  When a Fluent Bit service is defined in
+ * the environment, that will be used.  Alternatively, if in a Cloud
+ * environment, logging directly to CloudWatch will be performed. This is not
+ * recommended as it can cause server errors when CloudWatch is rate limiting
+ * requests or the service itself is down.
+ *
+ * @param string $log_group Name of the log group to send logs to. Environment
+ *                          name is added automatically. For instance, specifying
+ *                          'foobar' in an environment named some-client-prod-01
+ *                          will log to the log group named
+ *                          some-client-prod-01/foobar. The log group is
+ *                          expected to already exist in AWS; the log group
+ *                          will not be created if it does not already exist.
+ * @param string $log_stream Name of the log stream. This value is used as
+ *                           provided.
+ * @return Psr\Log\LoggerInterface
+ */
+function get_logger( string $log_group, string $log_stream ) : LoggerInterface {
+	// $tag_name is designed to be used with Fluent Bit and doubles as a nice
+	// index for storing our loggers in. Tags must be prefixed with 'app.' to be
+	// correctly routed by our Fluent Bit container. Fluent Bit is configured
+	// to extract the log group and log stream from this string, with the log
+	// group in the second position and the log stream in the third,
+	// deliminated by a period. It will automatically prepend the environment
+	// name to the log group.
+	$tag_name = sprintf( 'app.%s.%s', $log_group, $log_stream );
+
+	// Let's store each logger in an array so that we don't keep instantiating
+	// loggers. We create a new logger for each Monolog channel. The channel
+	// name will be used as the Fluent Bit tag.
```

Comment:
The commenting here is a bit too verbose IMO, and makes the code less readable. The first comment is great in terms of explaining _why_, but some of these just explain _what_, which you can already see from the code. Recommend slimming these down.

```suggestion
```